app/presenters/summary/components/investment.rb

```diff
@@ -0,0 +1,35 @@
+module Summary
+  module Components
+    class Investment < BaseRecord
+      alias investment record
+
+      private
+
+      def answers
+        [
+          Components::FreeTextAnswer.new(
+            :description, investment.description
+          ),
+          Components::MoneyAnswer.new(
+            :value, investment.value
+          ),
+          Components::ValueAnswer.new(
```

Comment:
If I'm understanding them correctly, designs are saying to leave this row out if partner == no

Reply:
Yes, I'm going to talk to design about this post hoc. I'm planning to extract the frontend side of ownership stuff in the National Savings Certificate feature so will address this then.